src/main/java/homework11/FruitShop.java

```diff
@@ -0,0 +1,151 @@
+package homework11;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.type.TypeFactory;
+import org.apache.log4j.Logger;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.security.SecureRandom;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class FruitShop {
+    private final static Logger logger = Logger.getLogger(FruitShop.class.getName());
```

Comment:
Naming is not right. 
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names